test/tstDistributedTree.cpp

```diff
@@ -185,12 +185,11 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(empty_tree, DeviceType, ARBORX_DEVICE_TYPES)
   }
   else
   {
-    ARBORX_TEST_QUERY_TREE(
-        ExecutionSpace{}, tree,
-        makeIntersectsSphereQueries<DeviceType>({
-            {{{(double)comm_rank, 0., 0.}}, (double)comm_size},
-        }),
-        make_reference_solution<PairIndexRank>({}, {0, 0}));
+    ARBORX_TEST_QUERY_TREE(ExecutionSpace{}, tree,
+                           makeIntersectsSphereQueries<DeviceType>({
+                               {{{(float)comm_rank, 0., 0.}}, (float)comm_size},
```

Comment:
If the types need to match, I would have expected that you also need to use `0.f` instead of `0.` (and I would prefer it for consistency).
```suggestion
                               {{{(float)comm_rank, 0.f, 0.f}}, (float)comm_size},
```